keras/layers/recurrent.py

```diff
@@ -300,19 +292,15 @@ class LSTM(Recurrent):
     '''
         Acts as a spatiotemporal projection,
         turning a sequence of vectors into a single vector.
-
         Eats inputs with shape:
         (nb_samples, max_sample_length (samples shorter than this are padded with zeros at the end), input_dim)
-
         and returns outputs with shape:
         if not return_sequences:
             (nb_samples, output_dim)
         if return_sequences:
             (nb_samples, max_sample_length, output_dim)
-
         For a step-by-step description of the algorithm, see:
         http://deeplearning.net/tutorial/lstm.html
-
```

Comment:
I'm fine with removing end-of-line extra spaces, but the spacing in the comments is here for a reason (readability). Can you exclude from this PR the files that are outside of the scope of the PR?